neurodamus/core/_engine.py

```diff
@@ -42,7 +42,7 @@ def find_plugins(cls):
         if plugin_module:
             importlib.import_module(plugin_module)
         # Auto import correctly named modules
-        for finder, name, ispkg in pkgutil.iter_modules():
+        for _finder, name, _ispkg in pkgutil.iter_modules():
```

Comment:
```suggestion
        for _, name, _ in pkgutil.iter_modules():
```

standard way to disregard elements of a tuple

Reply:
this way it's clear what they are, in case someone wants to use them